docs/logging-and-monitoring/health-metrics.md

```diff
@@ -0,0 +1,170 @@
+
+#### State: Synced
+
+On a healthy synced Node, the `rn_ledger_last_update_proposer_epoch_second` should always stay
+close to the wallclock, since a healthy Network always keeps committing to the Ledger (even in case
+of no "business" transactions, the consensus epoch/round updates are being committed - otherwise
+there must be a liveness break).
+
+A synced Node may either belong to an active Validator Set (and perform BFT commits), or receive
+timely Ledger sync updates.
+
+#### State: Out-of-sync
+
+On an unhealthy, out-of-sync Node, the `rn_ledger_last_update_proposer_epoch_second` keeps moving
+further and further into the past, despite the rest of the Network (most importantly: Validator Set)
+being healthy.
+
+Of course, a local connectivity loss can be hard to distinguish from the Network's liveness break,
+but some other metrics can narrow it down:
+
+- A non-progressing `rn_sync_target_proposer_timestamp_epoch_second` and `rn_sync_target_state_version`
+  are strong indicators of the Network's liveness problems (unless _all_ connected peers coordinated
+  to lie to the Node).
+- A growing `rn_sync_unexpected_responses_received_total` hints at network latency/timeout issues.
+- A growing `rn_sync_invalid_responses_received_total` (especially when caused by
+  `MISMATCHED_TRANSACTION_ROOT`) hints at malformed local Ledger state.
+- _(if the Node is in active Validator Set)_ A growing `rn_bft_rejected_consensus_events_total`
+  hints at byzantine behavior of the other Validators.
+
+### Consensus status
+
+The consensus health is harder to categorize into distinct states. It may be required to interpret
+each related metric individually: 
+
+rn_misc_wallclock_epoch_second
+: A current wallclock, as seen by the Node precisely at the moment of rendering the Prometheus
+endpoint response. All metrics which record timestamps (i.e. named `*_epoch_second`) should be
+interpreted as relative to this one. A Node with a seriously misconfigured clock will most
+likely not be a healthy Validator, since timestamps from BFT messages are verified against it.
+
+rn_misc_config_info
+: A Prometheus `info` metric containing a few items related to the Node's resolved configuration.
+
+The labels of `rn_misc_config_info` related to the Node's consensus health:
+- `post_genesis_epoch_state_hash`
+  A state hash captured after the execution of all the Genesis Transactions. This should be constant
+  throughout the Node's lifetime, and the same across the entire Network. 
+- `configured_validator_address`
+  A logical Validator component address (within the Engine). Only present (i.e. non-empty) if it 
+  was statically configured or resolved from Genesis.
+
+rn_bft_in_validator_set
+: A "boolean" gauge, showing `1.0` when this Node belongs to an active Validator Set of the
+_current_ epoch (or `0.0` otherwise).
+The `1.0` here is only possible when `rn_misc_config_info.configured_validator_address` is
```

Comment:
AFAIU from here, you can not have a full node and create the validator & "promote" it while it is running, right?

Reply:
That is my understanding as well (I've once heard that you have to restart a full node for your validator settings to be applied - is this still the case @LukasGasior1 ?)

Reply:
that's correct